File: codewars/Anagram difference/anagram.py
def anagram_difference(w1, w2):
    count = 0 
    len_w1 = len(w1)
    len_w2 = len(w2)
    if type(w1) == str:
        w1 = list(w1)
    if type(w2) == str:
        w2 = list(w2)
    for i in w1:
        if i in w2:
            count += 1
            w2.remove(i)
    print(w1, w2)
    return len_w1 - count + len_w2 - count

File: codewars/Anagram difference/Anagram.js
function anagramDifference(w1, w2) {
    const letterCount = {};
  
    for (let letter of w1) {
      letterCount[letter] = (letterCount[letter] || 0) + 1;
    }
  
    for (let letter of w2) {
      letterCount[letter] = (letterCount[letter] || 0) - 1;
    }
    let lettersToRemove = 0;
    for (let count of Object.values(letterCount)) {
      lettersToRemove += Math.abs(count);
    }
  
    return lettersToRemove;
  }
  